### DISCUSSION TITLE
Feature of lib

### DISCUSSION THREAD

**Discussion by Saleh-At** [whoenig/libMultiRobotPlanning#42] [General]
Hello there
I have some questions for your library!
• Can I use this in the Qt project? How to do that!
• Your algorithm is ...
> for 3D path finding or 2D?
> interested in only rover type as agent?
> how many agents can support?
> what have goals properties like x, y , z and delay time & ...?!
     >   if it supports delay, can remove it and replace with fixed or variable speed?!
> supported any size of agents?!
> supported velocity of agents?
> can visualize your result as animation?
🙏
Best regards

**Comment by whoenig** [comment]
> Can I use this in the Qt project? How to do that!

I haven't tried it myself, but I don't see a reason why not.

> for 3D path finding or 2D?

It's generic. The examples are for 2D.

> interested in only rover type as agent?

It's generic. The key assumption is typically constant velocity while moving on an edge.

> how many agents can support?

Tested with up to 200 or so, but depends on the difficulty of the environment mostly.

> what have goals properties like x, y , z and delay time & ...?!

Just position for the included examples.

> supported any size of agents?!

Only in the experimental mapf/c branch with additional code changes.

> supported velocity of agents?

No notion of velocity.

> can visualize your result as animation?

There is a Python script to visualize the 2D examples.
